LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright 2021 Tobias Falkner
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
```

Comment:
I'm not too familiar with the details of SW licensing myself neither but I think Apache 2.0 is fine and we don't need to mention the used packages as they're Apache 2.0 as well.

HOF did some investigation before publishing the CJS utils based on which we decided to use Apache 2.0. Details about what he found out is documented [in Notion](https://www.notion.so/combeenation/Open-Source-Licences-d4f78bd432ef4f4a9115daf2fe4cf6a9#bc72930968d04d75bb42d9728235f9e4).